roles/utils/hdfs_user_homes/tasks/main.yml

```
@@ -17,3 +17,5 @@
   become: true
   become_user: "{{ hdfs_user }}"
   loop: "{{ hdfs_user_homes }}"
+  loop_control:
+    label: "{'user': '{{ item.user }}', 'group': '{{ item.group }}', 'uid': '{{ item.uid }}', 'gid': '{{ item.gid }}'}"
```

Only `item.user` is mandatory inside `hdfs_user_homes` array, you should only display it.
```suggestion
    label: "{'user': '{{ item.user }}'}"
```